python_scripts/01_tabular_data_exploration.py

```diff
@@ -245,7 +245,7 @@
     x="age",
     y="hours-per-week",
     data=adult_census[:n_samples_to_plot],
-    hue="class",
+    hue=target_column,
```

Comment:
Could you please make the same modification in the scatterplot just below this one (line 269)?

Reply:
Done, thanks @ArturoAmorQ !